core/src/components/input/input.ionic.scss

```diff
@@ -20,6 +20,13 @@
   min-height: 48px;
 }
 
+// Ionic Input Shapes
+// --------------------------------------------------
+
+:host(.input-shape-round) {
+  --border-radius: 100px;
```

Comment:
I believe design is intending for this to use the `$ionic-border-radius-rounded-full` token. 

Would be good to confirm. I had discussed in the past with them about getting that updated from `100%` to `999px` since browsers will render them differently at larger scales.

Reply:
I just went ahead and changed it; as you alluded to, 100% is pretty obviously incorrect, even in Chrome:
![image](https://github.com/ionic-team/ionic-framework/assets/90629384/5dc47a65-0446-4e39-9ce3-1c749b6329e7)
The token also isn't currently used anywhere else, so it doesn't seem worth raising a fuss over.

Styles updated to use token in https://github.com/ionic-team/ionic-framework/pull/29329/commits/9ab05275c54a38ebc160da6369d84e9da7d10157, token value changed in https://github.com/ionic-team/ionic-framework/pull/29329/commits/275046054c8fdd2e59ef2b66676a08cbb6dac925.

Reply:
Yeah it should be `999px` - I noted this in some of the design documents and I know the UX requirements have it in a few places.